src/index.js

```diff
@@ -1,15 +1,14 @@
-/* eslint-disable import/no-mutable-exports */
-/* eslint-disable import/prefer-default-export */
 /* eslint-disable import/no-cycle */
 import './style.css';
```

Comment:
- [ ] As mentioned by the previous reviewer, Kindly make sure you fix all linters errors instead of disabling them.